### ISSUE TITLE
maybe a better name: node-sudoc

### ISSUE THREAD

**Issue by kerphi** [ezpaarse-project/sudoc#1]
so we understand easier that it is a nodejs binding for sudoc services
no ?

**Comment by nojhamster** [comment]
Makes sense, I don't mind as long as the npm package doesn't have the prefix. Since I haven't commited anything yet, I will recreate the repo tomorrow to change the name and add a MIT license.

**Comment by kerphi** [comment]
Maybe the npm package name could be sudoc and the github repository name
could be node-sudoc
However, I don't really know if its a common way to name nodejs packages.

**Comment by nojhamster** [comment]
Actually having **node** or **js** in the name of an NPM package doesn't really make sense because we are in a node environment. That's why github projects like `node-github`, `node-twitter` or `dropbox-js`  become `github`, `twitter` and `dropbox` in the npm environment.

If we need to make a clear distinction between the sudoc services and the node wrapper, maybe we should go for something like **sudoc-wrapper**, but I don't think it's necessary.

**Comment by kerphi** [comment]
My comment concerns renaming the "github repository" name but not the
nodejs npm package name.
I think it could be good to have:
github repo name = node-sudoc
npm package name = sudoc

**Comment by nojhamster** [comment]
My bad, I misunderstood. Then let's go for node-sudoc and sudoc :)

**Comment by pseudom** [comment]
I suggest meta-sudoc, but may be i missed something ! ;-)

**Comment by kerphi** [comment]
Thanks dom, I think that it's us who are wrong.
We have to name things coherently.

**Comment by nojhamster** [comment]
imho, I don't think the "meta" prefix makes sense for a wrapper. I even think that meta-doi should be renamed "crossref" once it's not limited to dois, which would be a more meaningful name.

**Comment by kerphi** [comment]
yes but the name crossref alone whould have the same problem as sudoc alone.
crossref alone could be a lot of things about crossref :)
so node-* everywhere would be better for these nodejs wrappers ?

**Comment by nojhamster** [comment]
I think so, always in the context of github of course.

**Comment by kerphi** [comment]
It makes sense for me.
